### ISSUE TITLE
请问如何获取卷积（算子）中的配置参数，例如pytorch中可以通过conv.in_channels获取输入通道设置

### ISSUE THREAD

**Issue by d2623587501** [PaddlePaddle/Paddle#44100]
### 请提出你的问题 Please ask your question

代码如下，尝试获取输入卷积配置参数，无法获取
`def fuse(conv, bn):
    print(conv.parameters)
    fused = paddle.nn.Conv2D(
        conv.in_channels,
        conv.out_channels,
        kernel_size=conv.kernel_size,
        stride=conv.stride,
        padding=conv.padding,
        bias=True
    )`

**Comment by Aganlengzi** [comment]
@d2623587501 你好，parameters获取到的是训练参数，比如weight和bias，看你的描述是想问Paddle中paddle.nn.Conv2D是否支持传入参数(in_channels, out_channels...)等的获取是吗？目前Paddle可通过`_xxx`的方式获取。

paddle:
<img width="664" alt="93ad5f30bc3a542bc44530c0186a0a2c" src="https://user-images.githubusercontent.com/5997715/177308587-2721b403-2956-4afb-8619-e68ae166256c.png">
<img width="591" alt="image" src="https://user-images.githubusercontent.com/5997715/177319384-a67121cd-ff4d-4477-ba20-3fea0125593c.png">


torch:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/5997715/177308547-5b69b9f0-ebea-4857-b102-afebecd072cd.png">

**Comment by d2623587501** [comment]
好滴  感谢大佬的回复

**Comment by Aganlengzi** [comment]
@d2623587501 Update，您好，咨询了相关同学，目前Paddle支持以_形式获取输入参数，请您试用。

**Comment by d2623587501** [comment]
谢谢大佬  通过参数前添加_问题解决了， 如（conv._in_channels），再次感谢
